packages/bun-usockets/src/socket.c

```diff
@@ -348,8 +351,10 @@ int us_socket_write(int ssl, struct us_socket_t *s, const char *data, int length
 
     int written = bsd_send(us_poll_fd(&s->p), data, length, msg_more);
     if (written != length) {
-        s->context->loop->data.last_write_failed = 1;
-        us_poll_change(&s->p, s->context->loop, LIBUS_SOCKET_READABLE | LIBUS_SOCKET_WRITABLE);
+        if(s->context->loop) {
```

Comment:
Is this a thing that happens? It has to be undefined memory right?

Reply:
forgot to undo this actually, this will never happen